### PR TITLE
[T3A5][W15-A3]Marcus Ng Wen Jian

### PR DESCRIPTION
**Pull request by marcusngwj**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -75,7 +57,7 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        out.print(Formatter.LINE_PREFIX + "Enter command: ");
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/TextUi.java]: errr.... here the UI is still doing the string processing/formatting (particularly code inside the `print()`)

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -148,7 +130,7 @@ private void showToUserAsIndexedList(List<String> list) {
     /** Formats a list of strings as a viewable indexed list. */
     private static String getIndexedListForViewing(List<String> listItems) {
         final StringBuilder formatted = new StringBuilder();
-        int displayIndex = 0 + DISPLAYED_INDEX_OFFSET;
+        int displayIndex = 0 + Formatter.DISPLAYED_INDEX_OFFSET;
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/TextUi.java]: i think this could also be moved into the formatter,

**Comment by okkhoy** [comment]
@marcusngwj 
Some comments added. Please ack & close the PR after reading comments.

**Comment by marcusngwj** [comment]
Read. Thank you for the advice

**Pull request closed without merging**